### PR TITLE
Added SolarWinds Orion fingerprints

### PR DESCRIPTION
**Pull request by scopedsecurity**
## Description
Added SolarWinds Orion fingerprints in html_title.xml and favicons.xml

### CODE REVIEW

File: xml/favicons.xml [hunk 1]
```diff
@@ -196,6 +196,14 @@
     <param pos="0" name="service.certainty" value="0.5"/>
   </fingerprint>
 
+  <fingerprint pattern="^53317933c27890ae9218697ecc0e97d9$">
+    <description>SolarWinds Orion</description>
+    <example>53317933c27890ae9218697ecc0e97d9</example>
+    <param pos="0" name="service.vendor" value="SolarWinds"/>
+    <param pos="0" name="service.product" value="Orion"/>
+    <param pos="0" name="service.certainty" value="0.5"/>
```
Comment (rkirk-r7) [inline xml/favicons.xml:204]: Should we add the CPE for both of these fingerprints? 
`<param pos="0" name="service.cpe23" value="cpe:/a:solarwinds:orion_platform:-"/>`

Comment (scopedsecurity) [reply xml/favicons.xml:204]: I've updated both with the CPEs.

**Review by rkirk-r7 — APPROVED** [review]

**Comment by tsellers-r7** [comment]
I'll try to land this later today.  I expect that the automatic CPE generation ( https://github.com/rapid7/recog/blob/master/CONTRIBUTING.md#updating-cpes ) will fail so we will either change the name of the product from `Orion` to `Orion Platform` or add a mapping here:

https://github.com/rapid7/recog/blob/master/cpe-remap.yaml

**Comment by tsellers-r7** [comment]
@scopedsecurity - Landing this now, thanks for the contribution. As a note, I'll likely change `Orion` to `Orion Platform` to match the CPE.

**Pull request merged**